src/main.rs

```diff
@@ -349,6 +349,13 @@ fn run() -> Result<(), failure::Error> {
                         .short("e")
                         .long("env")
                         .takes_value(true)
+                )
+                .arg(
+                    Arg::with_name("print-files")
+                        .short("p")
```

Comment:
I think only commonly used flags should have a short one, `--p` could be something else in the future.

Reply:
Is there any reason we are not calling this `--verbose`? `--print-files` seems a bit funky to me. Not a gigantic dealio to me though

Reply:
I suggested it because it does only print files. People will probably expect to have a verbose on all commands where this will only be for Worker sites.